include/libcyphal/transport/errors.hpp

```diff
@@ -64,4 +75,17 @@ using MediaError = cetl::variant<ArgumentError, PlatformError, CapacityError>;
 }  // namespace transport
 }  // namespace libcyphal
 
+namespace cetl
+{
+
+// C6271889-BCF8-43A9-8D79-FA64FC3EFD93
+template <>
+constexpr type_id type_id_getter<libcyphal::transport::AnyError>() noexcept
+{
+    // NOLINTNEXTLINE(cppcoreguidelines-avoid-magic-numbers, readability-magic-numbers)
+    return {0xC6, 0x27, 0x18, 0x89, 0xBC, 0xF8, 0x43, 0xA9, 0x8D, 0x79, 0xFA, 0x64, 0xFC, 0x3E, 0xFD, 0x93};
+}
```

Comment:
We put it inside of the `MaybeError` (which is `unbounded_variant`) - hence the type id stuff.